scripts/perf-test/ec2-terminate.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+### BEGIN INIT INFO
+# Provides: ec2-terminate
+# Required-Start: $network $syslog
+# Required-Stop:
+# Default-Start:
+# Default-Stop: 0 1 6
+# Short-Description: ec2-terminate
+# Description: send termination email
+### END INIT INFO
+#
+
+case "$1" in
+    start|status)
+        exit 0
+        ;;
+    stop)
+        # run the below
+        ;;
+    *)
+        exit 1
+        ;;
+esac
+
+# send instance-terminated message
+# http://rogueleaderr.com/post/48795010760/how-to-notifyemail-yourself-when-an-ec2-instance/amp
```

Comment:
:question: Is this the source of this code? Probably want to make it more obvious that's what this link is for, e.g. 
```
# Code take from
```